packages/cli/src/utils.ts

```diff
@@ -197,9 +197,9 @@ const validatePath = async (value: string) => {
   return true;
 };
 
-export const inquireInstancePath = async (initialPath?: string) => {
+export const inquireInstancePath = async (initialPath?: string, skipValidation?: boolean) => {
```

Comment:
the naming is not that clear, we use `skipValidatation` here but `skipCoreCheck` in other places. Need to align them and also some comments could help reviewers to understand

Reply:
Thanks, good point indeed